### ISSUE TITLE
Scroll position not reset to top on re-render with new HTML on Android

### ISSUE THREAD

**Issue by timraybould** [react-native-community/react-native-webview#709]
**Bug description:**
My WebView's source is html, via a variable maintained in state. On Android only, changing state, causing a re-render with new HTML maintains the scroll position when it should be reset to the top of the new page.

**To Reproduce:**
* Set WebView source equal to {html: someStateVariable}
* View page on Android and scroll some down the page
* Change someStateVariable with different HTML

**Expected behavior:**
Expecting the webview scroll position to reset to top of page once it gets re-rendered with new HTML, as it does on iOS and how it does on both iOS and Android using the webview within react-native core.

**Environment:**
 - OS: Android
 - react-native version: 0.59.8
 - react-native-webview version: 5.12.1

**Comment by timraybould** [comment]
This "bug" was introduced via the fix at #485. The loadData function in the Android SDK has its issues but it does properly reset the page to the top each time there is a change in the source. The loadDataWithBaseURL function does not.

One half-fix is to add either `view.scrollTo(0,0);` (abrupt) or `view.PageUp(true)` (with a scroll animation) right after calling `view.loadDataWithBaseURL` within RNCWebViewManager.java. However, it's not perfect because you can see the scrolling happening instead of just the page changing.

I'm preferring PageUp since the delay created by the animation somewhat hides what's going on and feels more like part of the new page animating in. It's still not as good as what happened when `loadData' was being used, however, I agree it's not worth reverting back to that because #485 did solve other problems.

**Comment by fallaciousreasoning** [comment]
My workaround for this is to wrap the WebView in an array:

```jsx
{[<WebView
      key={somethingThatChangesWhenYourContentChanges}
      source={{ html: html }}/>]}
```

which works because it tricks React into using a new WebView.
